### PR TITLE
Add a force-able builder for coverage?

### PR DESCRIPTION
**Pull request by kshyatt**
Hopefully I did this right?

### CODE REVIEW

File: master/coverage.py [hunk 1]
```diff
@@ -7,6 +7,12 @@
 julia_coverage_scheduler = Triggerable(name="Julia Coverage Testing", builderNames=julia_coverage_builders)
 c['schedulers'].append(julia_coverage_scheduler)
 
+c['schedulers'].append(ForceScheduler(
+    name="coverage build",
+    builderNames=julia_coverage_builders,
+    ]
```
Comment (tkelman) [inline master/coverage.py]: this looks mismatched?

Comment (kshyatt) [inline master/coverage.py]: Oh damn. Sorry, will fix.

**Comment by staticfloat** [comment]
Welp, let's see if this works!  I'm thinking we have to supply a commit gitsha somewhere, but we can always try it out!

**Comment by tkelman** [comment]
kinda?

image

**Comment by staticfloat** [comment]
How about now?

**Comment by tkelman** [comment]
yep, looks way better now, will leave it up to @kshyatt to take it for a test ride though

image

**Comment by staticfloat** [comment]
I just realized that this builder doesn't use the `revision` property at
all.  Improvements in progress!
-E

**Comment by staticfloat** [comment]
Alright, it now has a "url" field which defaults to the latest binary, but can be manually pointed at any `x86_64` binary on our AWS server to run against that particular commit.  Care to try it out, @kshyatt?

**Comment by kshyatt** [comment]
I pressed the button!!

**Pull request merged**